samples/Styleguide/src/Project/Styleguide/client/Components/Styleguide-Layout.razor

```diff
@@ -10,18 +10,18 @@
         <Placeholder Name="jss-styleguide-layout" Rendering="@Component" />
     </div>
     <div class="col-sm-4 col-lg-2 order-sm-first pt-2">
-        @foreach (var section in _sections)
+        @foreach (var s in _sections)
```

Comment:
Please revert these variable name changes.

Reply:
Might just be my environment, but my VS2019 is interpreting line 15 as [razor @section directives](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/layout?view=aspnetcore-3.1#sections) and I get the errors: The 'section` directive must appear at the start of the line; The 'section' directives value(s) must be separated by whitespace.